src/style.css

```diff
@@ -1 +1,74 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --text-color: #3d4e61;
+  --border-color: #e1e7ed;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.search__container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.search_item {
+  margin-top: 10px;
```

Comment:
it's margin collapsing. 
You should move margin-top to a container and check values to try pass tests. 
Check paddings, width, height for elements at figma